entries/QUnit.testDone.xml

```diff
@@ -26,14 +26,17 @@
 			<property name="runtime" type="Number">
 				<desc>The total runtime in millseconds of the test, including beforeEach and afterEach</desc>
 			</property>
+			<property name="skipped" type="Boolean">
+				<desc>Indicate whether or not the current test was skipped</desc>
+			</property>
 		</argument>
 	</signature>
 	<desc>Register a callback to fire whenever a test ends.</desc>
 	<example>
 		<desc>Register a callback that logs results of a single test</desc>
 		<code><![CDATA[
 QUnit.testDone(function( details ) {
-	console.log( "Finished running: ", details.module, details.name, "Failed/total: ", details.failed, details.total, details.duration );
+	console.log( "Finished running: ", details.module, details.name, "Failed/total: ", details.failed, details.total, details.duration, details.skipped );
```

Comment:
`details.skipped` here is a Boolean which indicates whether the test was skipped or not. But, I'm open to suggestion to change this test to something better.